ts/Extensions/Annotations/Popup.ts

```diff
@@ -872,55 +877,116 @@ H.Popup.prototype = {
                 callback
             );
         },
+
         /**
-         * Sort the series in the object.
+         * Filter object of series which are not indicators.
+         * If the filter string exists, check against it.
+         *
          * @private
+         *
+         * @param {Highcharts.FilteredSeries} series
+         *        All series are available in the plotOptions.
+         *
+         * @param {string|undefined} filter
+         *        Applied filter string from the input.
+         *        For the first iteration, it's an empty string.
+         *
+         * @return {void}
          */
-        sortSeries: function (series: Highcharts.FilteredSeries): Highcharts.FilteredSeries {
-            let sorted: Highcharts.FilteredSeries = {},
-                array = [],
-                key;
-
-            for (key in series) {
-                if (Object.prototype.hasOwnProperty.call(series, key)) {
-                    array.push(key);
-                }
-            }
+        filterSeries: function (
+            this: Highcharts.Popup,
+            series: Highcharts.FilteredSeries,
+            filter?: string
+        ): [[string, string, Series]] {
```

Comment:
Would an object be better than the array in the array?

Reply:
We've discussed that with @pawelfus and there are two main reasons why I chose array:
1. Avoiding looping through an object multiple times.
2. In the case of the `EDIT` bookmark, series are placed in the array. With array here, we can use one logic to handle both cases.

Later in the code, I've tried to give some descriptive names for each element in the array.
`[indicatorFullName, indicatorType, series]`

Reply:
However, an array of objects looks a bit better.